dags/xlml/solutionsTeam_pax_latest_supported.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A DAG to run all supported ML models with the latest PAX version."""
+
+import datetime
+from airflow import models
+from configs import composer_env, gcs_bucket, vm_resource
+from configs.xlml.jax import solutionsTeam_pax_latest_supported_config as pax_config
+
+
+# Run once a day at 10 pm UTC (2 pm PST)
+SCHEDULED_TIME = "0 10 * * *" if composer_env.is_prod_env() else None
+
+
+with models.DAG(
+    dag_id="pax_latest_supported",
+    schedule=SCHEDULED_TIME,
+    tags=["solutions_team", "pax", "latest", "supported", "xlml"],
+    start_date=datetime.datetime(2023, 11, 6),
```

Comment:
should we update this date to 11/8?

Reply:
Sure thing. I updated to 11-8, but it may still have some discrepancy between date of creating PR and that of the merge.